src/demos/storage-access-api/personalization.ejs

```diff
@@ -0,0 +1,76 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const baseURL = '<%= protocol %>://<%= domainC %><% if (isPortPresent) { %>:<%= port %><% } %>/personalization';
+    const pageContainer = document.getElementById('theme-container');
+    const themeSwitcher = document.getElementById('dark-mode-switch');
+    
+    async function updateUserPreference() {
+
+        const hasStorageAccess = await document.hasStorageAccess();
+        
+        if ( hasStorageAccess ) {
+            fetchAndApplyTheme();
+        } else {
+            try {
+                await document.requestStorageAccess();
+                fetchAndApplyTheme();
+            } catch (error) {
+                console.error('There has been a problem with your fetch operation:', error);
+            }
+        }   
+    }
+
+    function fetchAndApplyTheme() {
+        fetch(`${baseURL}/get-personalization`, {
+                method: 'GET',
+                credentials: 'include'
+            })
+                .then(response => {
+                    if (!response.ok) {
+                        throw new Error('Network response was not ok');
+                    }
+                    return response.json();
+                })
+                .then(data => {
+                    const theme = data.theme;
+                    pageContainer.className = theme;
```

Comment:
Doing this replaces all existing classes of that container, you'll need to add the new class, rather than replacing it

Reply:
I have solve it on the latest PR